os/kernel/binary_manager/binary_manager.c

```diff
@@ -138,6 +139,13 @@ int binary_manager(int argc, char *argv[])
 		bmdbg("Failed to open message queue\n");
 		return 0;
 	}
+	/* Create manager queue for fault handler */
+	gbinmgr_mq = mq_open(BINMGR_REQUEST_MQ, O_WRONLY, 0666, &attr);
```

Comment:
I think it is better to open mq with O_RDWR for binmgr_mq_fd  without this.

Reply:
yes, used common fd for both read and write

Reply:
yes, changed to single fd for read and write